core/domain/classifier_services_test.py

```diff
@@ -69,17 +72,41 @@ def _init_classify_inputs(self, exploration_id):
     def _create_classifier_training_job(
             self, algorithm_id, interaction_id, exp_id, exp_version,
             next_scheduled_check_time, training_data, state_name, status,
-            classifier_data, data_schema_version):
+            classifier_data, algorithm_version):
         """Creates a new classifier training job model and stores
         classfier data in a file.
         """
         job_id = classifier_models.ClassifierTrainingJobModel.create(
             algorithm_id, interaction_id, exp_id, exp_version,
             next_scheduled_check_time, training_data, state_name, status,
-            data_schema_version)
-        fs_services.save_classifier_data(exp_id, job_id, classifier_data)
+            algorithm_version)
+        classifier_data_proto = text_classifier_pb2.TextClassifierFrozenModel()
+        classifier_data_proto.model_json = json.dumps(classifier_data)
+        fs_services.save_classifier_data(exp_id, job_id, classifier_data_proto)
         return job_id
 
+    def _get_classifier_data_from_classifier_training_job(
+            self, classifier_training_job):
+        """Retrieves classifier training job from GCS using metadata stored in
+        classifier_training_job.
+
+        Args:
+            classifier_training_job: ClassifierTrainingJob. Domain object
+                containing metadata of the training job which is used to
+                retrieve the trained model.
+
+        Returns:
+            FronzeModel. Protobuf object containing classifier data.
+        """
+        filepath = classifier_training_job.classifier_data_file_name
```

Comment:
Same question as before about filepath vs fillename.

Should this be stored in a common place rather than duplicating, btw? Like BaseClassifierTestClass?

Reply:
Its required only at two places (and I don't think it will be required at any other place even in future). What do you suggest we should do?

Reply:
Probably still refactor to a common method. We might write more tests going forward.

Reply:
Done